kubernetes/chart/templates/ckcore/deployment.yaml

```diff
@@ -48,6 +48,8 @@ spec:
             - {{ .Values.ckcore.graphdb.database | quote }}
             - --graphdb-username
             - {{ .Values.ckcore.graphdb.login | quote }}
+            - --psk
```

Comment:
Sorry I meant, the value should be handed in via the environment using the `PSK` env var. The `--psk` arg is just the equivalent to the env var.
Two reasons for this; for one if it's handed in via `--psk` then the secret is visible in the host's process list to anyone with access to it. The other, more important reason, when using `cksh` via the container then there's a special check for `PSK` from the env and a shim is used that provides that PSK to `cksh` transparently. When not handed in via the env  then the user would have to provide `--psk` themselves anytime they call `cksh`.